nb/projects/pentacular/28BYJ-48/28BYJ-48.nb

```diff
@@ -1,56 +1,118 @@
-export const bodyDiameter = 28;
-export const bodyHeight = 19;
-export const bodyPlay = 0.1;
-export const totalDiameter = 42;
-export const tabDiameter = 7;
-export const tabThickness = 1;
-export const holeDiameter = 4;
-export const shaftDiameter = 5;
-export const shaftFlats = 3;
-export const shaftFlatsLength = 6;
-export const shaftOffset = 6;
-export const shaftBaseDiameter = 9;
-export const shaftBaseHeight = 2;
-export const shaftLength = 10;
-export const shaftPlay = 0.2;
-export const connectorWidth = 15;
-export const connectorLength = 7;
-export const connectorBaseWidth = 17;
-export const connectorBaseHeight = 4;
-export const connectorHeight = 5;
-export const reelDiameter = 20;
-export const reelThickness = 4;
+import { FlatShaft } from 'https://raw.githubusercontent.com/jsxcad/JSxCAD/master/nb/components/shaft.nb';
 
-export const stepperMotor28byj48 = Group(
-  Arc(bodyDiameter)
-    .add(Box(connectorLength, connectorWidth).x(bodyDiameter / -2))
-    .add(Box(connectorBaseHeight, connectorBaseWidth).x(bodyDiameter / -2 + 2))
-    .ez(-bodyHeight)
-    .mask(grow(bodyPlay))
-    .as('body'),
+md`
+### 28BYJ-48
```

Comment:
How do users find these? Like if someone want's to use a 28BYJ-48 motor in their project, how do they know that it is an option to import one?

Reply:
That I'm not sure -- I've started having the parametric examples generate example code like:

"import ...; Foo(...)"

But I don't have a good idea for discovery.

Reply:
Is it possible to scrape github somehow? Maybe with a tag?

Reply:
Hmm, could be -- how do github tags work?

Reply:
Ah I think what I really meant is "topics" https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics

Basically it is an easy to way to group together all the repositories on a topic so if we used the 'jot' topic then we could search or browse within that topic

Reply:
Ah, right -- that's a good idea.

Is there any finer granularity available or just at the repository level?

Reply:
> also if the repo has jot topic, it would be wise to define markup for discovery. for example putting something into package.json so scraper can know where stuff is

That's an interesting idea.

Are there any standards on that or do we just add something ad hoc?

Reply:
I know of no specific standards for package.json except the obvious that you want a unique key there, but the object value of your property is I think totally up-to you.